src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -219,11 +219,11 @@ export default {
         },
         addConditionSet() {
             let conditionSetDomainObject;
-            const handleItemSelection = (item) => {
```

Comment:
which rule was this violating, and why wasn't `dismissDialog` below violating it?